<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwapDialog</class>
 <widget class="QDialog" name="SwapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swap items</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Item to &amp;return:</string>
     </property>
     <property name="buddy">
      <cstring>input_returnItem_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="input_returnItem_lineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="input_lookup_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Lookup</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Item to r&amp;ent</string>
     </property>
     <property name="buddy">
      <cstring>input_rentItem_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="input_rentItem_lineEdit"/>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTextBrowser" name="output_contract_textBrowser"/>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>input_returnItem_lineEdit</tabstop>
  <tabstop>input_lookup_pushButton</tabstop>
  <tabstop>input_rentItem_lineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>output_contract_textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SwapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SwapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
